user-agent-parser.opam

```diff
@@ -9,15 +9,15 @@ bug-reports: "https://github.com/issuu/uap-ocaml/issues"
 build: [["dune" "build" "-p" name "-j" jobs]]
 run-test: [["dune" "runtest" "-p" name "-j" jobs]]
 depends: [
-  "alcotest" {with-test & >= "0.8.3" & < "0.9.0"}
```

Comment:
Pretty sure alcotest has an API breakage, maybe it will need adapting to the new version.

Reply:
The tests did seem to run fine.

Reply:
With Alcotest >= 1.0?

Reply:
Yes. I'll try again, but I ran `make test` without issues.

Reply:
Ah, okay, I see Travis also succeeding. Sorry for the false alarm, was burned once by Alcotest before.

Reply:
I get this kind of output:

```
$ make test
dune runtest --force
        test alias test/runtest
Testing `test suite'.
This run has ID `FDA33733-ADF0-4DB1-ABA3-113E16F73EEE'.

{
  "success": 17664,
  "failures": 0,
  "time": 5.000000
}
```

Reply:
Maybe you had problems because of the [breaking change in `alcotest-async`](https://github.com/mirage/alcotest/releases/tag/1.0.0).

Reply:
Yes, that's most likely it. Since we don't use that it is immaterial. Thanks.